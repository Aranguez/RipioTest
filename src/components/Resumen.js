import React from 'react';

const Resumen = props => {
    return (
        <div className="box">
            <span>Enviado a</span><br/>
            <span className="t-azul"><b>{ props.address }</b></span>
            <hr/>
            <p><b>Enviado</b> <span className="t-rojo">{ props.amount }</span></p>
            <p><b>Billetera</b> { props.totalAmount }</p>
        </div>
    );
}

export default Resumen;
