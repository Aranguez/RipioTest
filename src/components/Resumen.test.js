import React from 'react';
import { shallow } from 'enzyme';
import Resumen from './Resumen';

describe('resumen', () => {
    const resumen = shallow(<Resumen/>)
    
    it('renders properly ', () => {
        expect(resumen).toMatchSnapshot()
    });
})