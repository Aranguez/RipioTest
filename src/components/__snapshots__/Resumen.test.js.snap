// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resumen renders properly  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Resumen />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h4>
          Envio realizado a: 
        </h4>,
        <hr />,
        <p>
          Enviado: 
        </p>,
        <p>
          Total: 
        </p>,
      ],
      "className": "box",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Envio realizado a: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Envio realizado a: ",
          undefined,
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "hr",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Enviado: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Enviado: ",
          undefined,
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Total: ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Total: ",
          undefined,
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4>
            Envio realizado a: 
          </h4>,
          <hr />,
          <p>
            Enviado: 
          </p>,
          <p>
            Total: 
          </p>,
        ],
        "className": "box",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Envio realizado a: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Envio realizado a: ",
            undefined,
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "hr",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Enviado: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Enviado: ",
            undefined,
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Total: ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Total: ",
            undefined,
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
